pkg/oc/bootstrap/docker/dockerhelper/helper.go

```diff
@@ -157,6 +157,13 @@ func (h *Helper) CheckAndPull(image string, out io.Writer) error {
 	if err != nil {
 		return starterrors.NewError("error pulling Docker image %s", image).WithCause(err)
 	}
+
+	_, _, err = h.client.ImageInspectWithRaw(image, false)
```

Comment:
put a comment in about why we're doing this.

Reply:
Updated